jest.config.js

```diff
@@ -0,0 +1,6 @@
+const config = {
+    testTimeout: 42222,
```

Comment:
Why such a huge number? Please use something more reasonable. If it's in ms, something like:

```suggestion
    testTimeout: 4000,
```

Reply:
Right :) I did not think about it. Thank you for the suggestion.

Reply:
Having a lower number here also has the nice benefits that you could argue that slow tests indicate a performance bug, that needs fixing ;)

And if something is timing out, then CI/CD will fail faster.